agents/zmq/src/zmq_agent.cc

```diff
@@ -917,7 +917,7 @@ void ZmqAgent::got_env_metrics(const ThreadMetrics::MetricsStor& stor) {
 NSOLID_PROCESS_METRICS_NUMBERS(V)
 #undef V
 #define V(Type, CName, JSName, MType) \
-            , proc_stor.CName.c_str()
+            , json(proc_stor.CName).dump().c_str()
```

Comment:
Note to self: migrate all JSON serialization to use `node::JSONWriter` and compare performance to current implementation.